task.go

```diff
@@ -224,7 +231,12 @@ func (e *Executor) runDeps(ctx context.Context, t *taskfile.Task) error {
 		d := d
 
 		g.Go(func() error {
-			return e.RunTask(ctx, taskfile.Call{Task: d.Task, Vars: d.Vars})
+			err := e.RunTask(ctx, taskfile.Call{Task: d.Task, Vars: d.Vars})
+			if err != nil {
+				e.Logger.Errf("%s", err)
```

Comment:
This log isn't needed. If an error happen, it'll be returned and logged by the caller.